include/fastdds/rtps/builtin/data/ReaderProxyData.h

```diff
@@ -51,12 +52,14 @@ class ReaderProxyData
 
     RTPS_DllAPI ReaderProxyData(
             const size_t max_unicast_locators,
-            const size_t max_multicast_locators);
+            const size_t max_multicast_locators,
+            const ContentFilterProperty::AllocationConfiguration& content_filter_limits = {});
 
     RTPS_DllAPI ReaderProxyData(
             const size_t max_unicast_locators,
             const size_t max_multicast_locators,
-            const VariableLengthDataLimits& data_limits);
+            const VariableLengthDataLimits& data_limits,
+            const ContentFilterProperty::AllocationConfiguration& content_filter_limits);
```

Comment:
This constructor is also part of the public API. The new parameter should have a default value so API is not broken in the minor release.